src/ShopPaymentTransactionBundle/pkgShopOrderStatus/objects/TPkgShopPaymentTransaction_TPkgShopOrderStatusManager.class.php

```diff
@@ -44,7 +51,7 @@ protected function triggerTransactionBasedOnStatus(TdbShopOrderStatus $oStatus)
         $paymentHandler = $oOrder->GetPaymentHandler();
 
         if (false === ($paymentHandler instanceof \esono\pkgshoppaymenttransaction\PaymentHandlerWithTransactionSupportInterface)) {
-            throw new TPkgCmsException_Log('payment handler '.get_class($paymentHandler).' must implement \esono\pkgshoppaymenttransaction\PaymentHandlerWithTransactionSupportInterface',
+            throw new TPkgShopPaymentTransactionException_PaymentHandlerDoesNotSupportTransaction('payment handler '.get_class($paymentHandler).' must implement \esono\pkgshoppaymenttransaction\PaymentHandlerWithTransactionSupportInterface',
```

Comment:
-> "Payment handler .... does not support ..."

Reply:
ok